test/data/leo/extra_long_canvas.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Leo test</title>
+</head>
+<body>
+    <script>
+      for (let i = 0; i < 21; i++) {
+        const div = document.createElement('div');
+        div.style.width = '559px';
+        div.style.height = '794px';
+
+        const canvas = document.createElement('canvas');
+        const context = canvas.getContext('2d');
+        canvas.width = 559
+        canvas.height = 794
+        context.font = '30px Arial';
+        if (i == 19) {
+          // This should be the last page.
```

Comment:
// This should be the last page because of the truncation

Reply:
addressed and squashed.